docroot/modules/humsci/hs_blocks/templates/block--social-media.html.twig

```diff
@@ -0,0 +1,12 @@
+{% if links %}
+  <ul class="social-media-link__list">
```

Comment:
use `{{ attributes }}` to be consistent with other templates and allow additional attributes via hook functions.

Reply:
Fixed.